<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTester</class>
 <widget class="QWidget" name="QTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>GoTo State</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbStates">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Frequence (MHz)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFreq">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSet">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SpiNNaker</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSpin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Spin3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spin5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Chip</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbChip">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbCore">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>52</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>17</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPLL">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PLL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>173</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Read current PLL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
